src/js/tether.js

```diff
@@ -102,39 +101,26 @@ class TetherClass extends Evented {
 
     this.setOptions(options, false);
 
-    TetherBase.modules.forEach((module) => {
```

Comment:
We should make sure adding custom modules to Tether still works. I think it should, since they live on `Tether.modules` and not `TetherBase.modules`, but I do not see anything that uses those, so we should add tests for it http://tether.io/advanced/extending_tether/

Reply:
You're right, since it can be customized we need to run this on any potential module that could be added.